pkg/apis/pipeline/v1/task_validation_test.go

```diff
@@ -77,8 +91,9 @@ func TestTaskSpecValidate(t *testing.T) {
 		Results      []v1.TaskResult
 	}
 	tests := []struct {
-		name   string
```

Comment:
It might be cleaner to have one test class where you do validate params + workspaces, and one where you don't.